macros/finecalib_step0.C

```diff
@@ -0,0 +1,67 @@
+std::vector<int> devices_indices = {192, 193, 194, 195, 196, 197, 198, 207};
+
+TF1 *f_fit = new TF1("f_fit", "[0]*(1./(exp((x-[1])/[2])+1))*(1./(exp(([3]-x)/[4])+1))", 10., 200.);
+
+void finecalib_step0(std::string input_filename, std::string output_filename = "finetune_calib0.root", int reference_bin_for_distribution_heigth = 70, int minimum_entries = 1)
```

Comment:
output_filename = "finecalib_step0.root"

Reply:
remove the `reference_bin_for_distribution_height` from the parameters.
it can likely automatically found from the histogram because it is likely the same number as the `GetMaximumBin`